### ISSUE TITLE
Sponge problems with commands

### ISSUE THREAD

**Issue by Raycoms** [SpongePowered/SpongeForge#1361]
We have our own custom commands which already do the checks if the player is allowed to execute it.
First of all it checks for the op permission.

But it seems like if we put the command in the allowed list, sponge gives op level to that player for that command. Which screw some of our things up seriously.

**Comment by Raycoms** [comment]
Is there a fix for that?

**Comment by bloodmc** [comment]
Why not just use a permissions plugin like LuckPerms? The op system is terrible and there is no need to handle custom permission checks yourself.

**Comment by ryantheleach** [comment]
@Raycoms Do you have any elaboration? as it stands it's a little hard to follow what you mean.

**Comment by Raycoms** [comment]
Got it figured out, forgot to close this.
Thanks
